src/components/forms/form-changeavatar/form-changeavatar.ts

```diff
@@ -22,15 +22,19 @@ export class FormChangeAvatar extends Form {
       submit: async (e: Event) => {
         e.preventDefault();
         const submitMessage: HTMLElement = document.querySelector(
-          ".submit-message"
+          ".result-message"
         ) as HTMLElement;
-        const form: HTMLFormElement = document.querySelector(
-          "form.form__changeAvatar"
-        ) as HTMLFormElement;
+        const input: HTMLInputElement = document.querySelector(
+          "#avatar"
+        ) as HTMLInputElement;
```

Comment:
```suggestion
        const input = document.querySelector<HTMLInputElement>("#avatar")
```

Если можно избежать явного приведения типов, его лучше избегать

Reply:
О, спасибо, я и не знала, что так можно )